Sources/Features/Breadcrumb/BacktraceNotificationObserver.swift

```diff
@@ -0,0 +1,277 @@
+import Foundation
```

Comment:
why we don't remove observers? I couldn't find the code responsible for doing it.

Reply:
If you read the documentation on observers, it mentions that from iOS9 it is unregistered automatically:

```
If your app targets iOS 9.0 and later or macOS 10.11 and later, and you used [addObserver(_:selector:name:object:)](https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver), you do not need to unregister the observer. If you forget or are unable to remove the observer, the system cleans up the next time it would have posted to it.
```
https://developer.apple.com/documentation/foundation/notificationcenter/1413994-removeobserver

Reply:
If you check BacktraceBreadcrumbs it nils the BacktraceNotificationObserver. Garbage collection of iOS will take care of it. 
https://docs.swift.org/swift-book/LanguageGuide/Deinitialization.html

Regardless, I spotted the opportunity to change the collection to a constant.